x-pack/plugins/security_solution/server/lib/risk_engine/tasks/constants.ts

```diff
@@ -9,7 +9,7 @@ export const SCOPE = ['securitySolution'];
 export const TYPE = 'risk_engine:risk_scoring';
 export const VERSION = '0.0.1';
 export const INTERVAL = '1h';
-export const TIMEOUT = '5m';
+export const TIMEOUT = '10m';
```

Comment:
Hey, just to be sure, what does this timeout mean for the task manager?

Does it mean, that it will be canceled after 10m of doing work?

Reply:
Correct.